src/utils/db.rs

```diff
@@ -54,6 +56,7 @@ pub fn cutout2bsonbinary(cutout: Vec<u8>) -> mongodb::bson::Binary {
 
 // This function, for a given survey name (ZTF, LSST), will create
 // the required indexes on the alerts and alerts_aux collections
+#[instrument(level = INFO, skip(db), err)]
```

Comment:
does the `skip(db)` basically means "log all variables but those in the skip", vs the `skip_all` that means don't log the inputs? Just curious!